File: java/8.10/example1.java
import java.util.Scanner;

public class example1 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        Integer moreThan10 = 0;
        Integer equalTo10 = 0;
        for (Integer x = 0; x < 100; x++) {
            System.out.println("Enter a number");
            Integer number = Integer.parseInt(scanner.nextLine());
            if (number > 10) {
                moreThan10++;
            } else if (number == 10) {
                equalTo10++;
            }
        }
        System.out.println(moreThan10);
        System.out.println(equalTo10);
    }
}

File: python/8.10/example1.py
moreThan10 = 0 # Setting the value to 0
equalTo10 = 0 # Setting the value to 0
for x in range(0, 100): # Running a loop 100 times
    number = int(input("Enter a number")) # Settng the value to the interger of the value inputed
    if number > 10: # Checking if number is greater than 10
        moreThan10 = moreThan10 + 1 # Adding 1 to the variable
    elif number == 10: # Checking if number is equal to 10
        equalTo10 = equalTo10 + 1 # Adding one to the variable